assets/i18n/ast.json

```diff
@@ -15,8 +15,8 @@
 	"I18N_CREATE_EXPLORATION_UPLOAD": "Xubir",
 	"I18N_CREATE_NO_THANKS": "Non, gracies",
 	"I18N_CREATE_YES_PLEASE": "¡Sí, por favor!",
-	"I18N_DASHBOARD_MY_COLLECTIONS": "Les mios colecciones",
-	"I18N_DASHBOARD_MY_EXPLORATIONS": "Les mios esploraciones",
+	"I18N_DASHBOARD_COLLECTIONS": "Les mios colecciones",
```

Comment:
I will go through them. But yes, I think it we should look forward to automating this process. As the number of languages increases, it would be difficult do this every time :P.